### PR TITLE
Add properties

### PR DESCRIPTION
**Pull request by MaltsevaKsenia**

### CODE REVIEW

File: src/test/java/com/epam/repository/HotelDAOTest.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void buildHotel(){
         when(resultSet.getString("name")).thenReturn(testHotelName);
         when(resultSet.getString("city")).thenReturn(testHotelCity);
         when(resultSet.getString("country")).thenReturn(testHotelCountry);
-        when(resultSet.getInt("numberOfStars")).thenReturn(testHotelStars);
+        when(resultSet.getInt("number_Of_Stars")).thenReturn(testHotelStars);
```
Comment (goponenkoev) [inline src/test/java/com/epam/repository/HotelDAOTest.java]: It seems like Of & Starts with the first letter in upper case, is it okay?

**Review by goponenkoev — APPROVED** [review]
Please check my previous comment, but for the rest is okay

File: src/test/java/com/epam/repository/HotelDAOTest.java (hunk 1 above)
Comment (MaltsevaKsenia) [reply src/test/java/com/epam/repository/HotelDAOTest.java]: done

**Review by VNTee — APPROVED** [review]

**Review by grigiv — APPROVED** [review]

File: src/main/java/com/epam/controller/ReservationController.java [hunk 2]
```diff
@@ -19,21 +19,25 @@ public ReservationController(ReservationService reservationService) {
...
-    public String listReservations(ModelMap modelMap) {
-        List<Reservation> list = reservationService.listReservations();
-        modelMap.addAttribute("listReservation", list);
+    @GetMapping
+    public String testadmin() {
+        return "redirect:/reservation/1";
+    }
+
+    @GetMapping("/{id}")
+    public String listReservations(@PathVariable Integer id, ModelMap modelMap) {
```
Comment (ghost) [inline src/main/java/com/epam/controller/ReservationController.java]: Is it about properties? Please, remove this from the pull request and create another with this changes

File: src/main/java/com/epam/service/ReservationService.java [hunk 3]
```diff
@@ -43,8 +43,16 @@ Reservation getReservationById(Integer id) {
         }
     }
 
-    public List<Reservation> listReservations() {
-        return reservationDAO.listReservations();
+    public List<Reservation> listReservations(Integer page, Integer total) {
```
Comment (ghost) [inline src/main/java/com/epam/service/ReservationService.java]: I believe is not about properties...

**Review by ghost — CHANGES_REQUESTED** [review]
please, remove all changes which are not about properties from this pull request, and create other pull requests if necessary

**Review by goponenkoev — COMMENTED** [review]
Please update this pull request and merge changes from dev

**Pull request merged**